docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,411 @@
+Gluon 2022.1
+============
+
```

Comment:
Typically there would be a `Important  Notes` section at the beginnging.

https://gluon.readthedocs.io/en/latest/releases/v2021.1.html

Something like "Upgrades to v2022.1 and later releases are only supported from releases ..."